auth0_flutter/lib/src/web/auth0_flutter_plugin_stub.dart

```diff
@@ -30,4 +30,9 @@ class Auth0FlutterPlugin extends Auth0FlutterWebPlatform {
     throw UnsupportedError(
         'hasValidCredentials is only supported on the web platform');
   }
+
+  @override
+  Future<void> logout(final LogoutOptions? options) {
+    throw UnsupportedError('logout is only supported on the web platform');
+  }
```

Comment:
This is a little weird and one case where sharing the existing platform implementation might have been useful.